README.md

```diff
@@ -13,3 +13,26 @@ This guide provides a uniform way to structure your RiotJS code. Making it
 * easier to cache and serve bundles of code separately.
 
 This guide is inspired by [AngularJS Style Guide](https://github.com/johnpapa/angular-styleguide) by John Papa.
+
+## Table of Contents
+
+1. [Component / Tag names](#component-tag-names)
+
+## Component / Tag names
```

Comment:
Should each rule be a command? Will we manage? :)
For instance we could also write this one as "Use short, meaningful, spec compliant component names".

And another question: Do we use the word "component" and "tag" interchangeably in Riot context? 
I would say modules are even more generic. Like a data model or service is also a module. To me, modules and components are normally the same (Node uses the term "modules", Bower uses the term "components"). Components have more of a UI connotation to them. So to me, each Riot tag is a UI module / UI component, but the opposite is not automatically true.